app/services/samples/destroy_service.rb

```diff
@@ -21,7 +21,17 @@ def execute
     private
 
     def destroy_single
-      sample.destroy
+      sample_destroyed = sample.destroy
+
+      if sample_destroyed
+        @project.namespace.create_activity key: 'namespaces_project_namespace.samples.destroy',
+                                           owner: current_user,
+                                           parameters:
+                                            {
+                                              sample_name: sample.puid,
```

Comment:
**nitpick (blocking):** Can this be renamed to `sample_puid`

Reply:
Updated in 032cb1c08ee993dee5d18b74ef198b03bafe07b3